packages/iTwinUI-react/src/core/Slider/Slider.test.tsx

```diff
@@ -229,6 +469,25 @@ it('should take railContainerProps', () => {
   expect(railContainer.style.marginRight).toBe('8px');
 });
 
+it('should take railContainerProps (vertical)', () => {
```

Comment:
Not needed as it is not vertical orientation specific and is already tested.

Reply:
**[DONE]** Thanks, removed this test